uk.co.compucorp.civicrm.tasksassignments/js/test/filters/due_test.js

```diff
@@ -61,14 +108,66 @@ define([
       it('hides the result for "dateRange" in the past', function() {
         expect($filter('filterByDue')(mock, 'dateRange', pastDateRange).length).toBe(0);
       });
+
+      it('shows the result for "dateRange" if from and until dates are empty', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyDateRange).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if from date is empty and until date is in the future but less than due date', function() {
+        expect($filter('filterByDue')(mockPlusThree, 'dateRange', emptyFromDateAndUntilDateInTheFuture).length).toBe(0);
+      });
+
+      it('shows the result for "dateRange" if from date is empty and until date is in the future but equals due date', function() {
+        expect($filter('filterByDue')(mockPlusTwo, 'dateRange', emptyFromDateAndUntilDateInTheFuture).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if from date is empty and until date is in the future but bigger than due date', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyFromDateAndUntilDateInTheFuture).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if from date is empty and until date is in the past', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyFromDateAndUntilDateInPast).length).toBe(0);
+      });
+
+      it('shows the result for "dateRange" if from date is empty and until date is today', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyFromDateAndUntilDateIsToday).length).toBe(0);
+      });
+
+      it('shows the result for "dateRange" if until date is empty and from date is in the past', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyUntilDateAndFromDateInThePast).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if until date is empty and from date is today', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyUntilDateAndFromDateIsToday).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if until date is empty and from date is in the future but less than due date', function() {
+        expect($filter('filterByDue')(mockPlusThree, 'dateRange', emptyUntilDateAndFromDateInTheFuture).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if until date is empty and from date is in the future but equals due date', function() {
+        expect($filter('filterByDue')(mockPlusTwo, 'dateRange', emptyUntilDateAndFromDateInTheFuture).length).toBe(1);
+      });
+
+      it('shows the result for "dateRange" if until date is empty and from date is in the future but bigger than due date', function() {
+        expect($filter('filterByDue')(mock, 'dateRange', emptyUntilDateAndFromDateInTheFuture).length).toBe(0);
+      });
```

Comment:
Could you  please replace `shows the result for `, similar to my suggestion above, It will make `it()` blocks shorter and self explanatory.